google-cloud/docs/toc.json

```diff
@@ -122,6 +122,32 @@
         }
       ]
     },
+    {
```

Comment:
This configuration should not be added until the gem is released.

Reply:
Removed.